### PR TITLE
[WebNN] Enable npm unit tests

### PR DESCRIPTION
**Pull request by zesongw**
### Description
- Support more test cases for WebNN EP in suite-test-list.jsonc
- Add DISABLE_WEBNN flag in build.ts as preparing for WebNN EP release
- Add test option: '--webnn-device-type' in test-runner-args-cli.ts to support running WebNN 'gpu' deviceType
- Use Chrome Stable as default browser for WebNN testing to unblock the CI limitation.

### CODE REVIEW

**Comment by zesongw** [comment]
@fdwr, @guschmue, PTAL, thanks!

File: js/web/test/test-runner.ts [hunk 1]
```diff
@@ -243,8 +243,14 @@ export class ModelTestContext {
       this.initializing = true;
 
       const initStart = now();
-      const session = await initializeSession(
-          modelTest.modelUrl, modelTest.backend!, modelTest.ioBinding, profile, sessionOptions || {}, this.cache);
+      const session = modelTest.backend == 'webnn' ?
+          await initializeSession(
+              modelTest.modelUrl, testOptions?.webnnOptions || 'webnn', modelTest.ioBinding, profile,
+              testOptions?.sessionOptions || {} || {}, this.cache) :
+          await initializeSession(
```
Comment (fdwr) [inline js/web/test/test-runner.ts:250]: I confess ignorable here about what's going on, but it seems weird for `initializeSession` to be called twice, one right after the other. 🤨

**Review by fdwr — COMMENTED** [review]
Except my one comment, everything looks reasonable to naive me, but Guenther could give more informative feedback than I can (or maybe Yulong). Thanks Zesong.

File: js/web/test/test-runner.ts (hunk 1 above)
Comment (fdwr) [reply js/web/test/test-runner.ts]: Thanks - that looks clearer now.

**Review by fdwr — APPROVED** [review]

**Review by guschmue — APPROVED** [review]

**Comment by guschmue** [comment]
lint is nagging:
image

**Review by guschmue — APPROVED** [review]

**Pull request merged**